deliverables/ruby_monk_problems/restaurant.rb

```diff
@@ -4,6 +4,10 @@ def initialize(menu)
   end
 
   def cost(*orders)
-    # your code here
+    orders.inject(0) do |total_cost, order|
+      total_cost + order.keys.inject(0) {|cost, key| cost + @menu[key]*order[key] }
+    end
   end
 end
+#where did "key" come from?
+#still unsure about what @ is with hashes
```

Comment:
`@` means the variable is an **instance variable**.  It is a variable that is set on the *instance* and maintained internally.  

*Instance variables* are available to all instance methods (anything starting with `def functionname`) without having to be passed as *arguments*